Makefile

```diff
@@ -1,9 +1,9 @@
 DESTDIR =
 PREFIX  = /usr/local
 
-BASHDIR = /etc/bash_completion.d
+BASHDIR = $(PREFIX)/etc/bash_completion.d
 ZSHDIR  = /usr/share/zsh/vendor-completions
```

Comment:
I don't use zsh and I'm not sure if it will pick up anything in `/usr/local/share/zsh/vendor-completions`. I did a cursory google search and found other packages explicitly installing into `/usr/share/zsh/vendor-completions`, so it sounds like that's the right place, but someone who actually uses zsh is free to chime in.